actions/rules/action_state_to_metastate.go

```diff
@@ -361,9 +361,13 @@ func (act ActionStateToMetaState) onStateChange(newContext change.Detector, cont
 		return nil, nil, err
 	}
 	// next, check which boards are relevant for this WI.
-	groups, err := act.Db.WorkItemTypeGroups().List(act.Ctx, wi.SpaceID)
+	space, err := act.Db.Spaces().Load(act.Ctx, wi.SpaceID)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, errs.Wrap(err, "error loading space: "+err.Error())
```

Comment:
No need to stringify the error here. Please get rid of `err.Error()`. You've already wrapped this error.

Reply:
Fixed. feecf1c6516ea161c05a91182563ba6d822033bd

Reply:
feecf1c6516ea161c05a91182563ba6d822033bd

Reply:
The fix is fine. But maybe it makes sense to specify which space could not be loaded?